libs/feature/editor/src/lib/components/record-form/form-field/form-field-map-container/form-field-map-container.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FormFieldMapContainerComponent } from './form-field-map-container.component';
+
+describe('FormFieldMapContainerComponent', () => {
```

Comment:
To do: write tests for this component

Reply:
Maybe this could be moved to feature/editor/src/lib too, to split things a bit more.